test/util/nightwatch-commands/axeCheck.js

```diff
@@ -19,16 +19,16 @@ export function command(context, config, _callback) {
   }, [axeSource]);
 
   // Run axe checks and report
-  this.executeAsync((innerContext, done) => {
+  this.executeAsync((innerContext, rules, done) => {
```

Comment:
@U-DON To answer your question about what `executeAsync()`, the second argument is an array of arguments to be passed to the function in the first argument: http://nightwatchjs.org/api/#executeAsync
So in this case `rules` can be either
1. `this.gobals.rules` which should have the value `['section508', 'wcag2a']` if defined in the nightwatch config
or
2. default to `['section508']`

Here are docs: http://nightwatchjs.org/api/#executeAsync